### PR TITLE
Expressiveness check foundations

### PR DESCRIPTION
**Pull request by flbulgarelli**
First iteration of #22

### CODE REVIEW

File: spec/ExpressivenessSpec.hs [hunk 1]
```diff
@@ -0,0 +1,25 @@
+module ExpressivenessSpec (spec) where
+
+import           Test.Hspec
+import           Language.Mulang.Inspector.Expressiveness
+import           Language.Mulang.Parsers.Haskell (hs)
+
+spec :: Spec
+spec = do
+  describe "isWellWritten" $ do
+    it "dsdsad" $ do
```
Comment (faloi) [inline spec/ExpressivenessSpec.hs]: Fix this please.

File: inspections [hunk 2]
```diff
@@ -0,0 +1,28 @@
+declaresObject bindingPattern
```
Comment (faloi) [inline inspections]: Delete this.

File: src/Language/Mulang/Parsers/Prolog.hs [hunk 3]
```diff
@@ -8,17 +8,18 @@ import Language.Mulang
...
-          (Right v) -> v
-          (Left m)  -> error.show $ m
+pl = orFail . parseProlog'
+
+parseProlog :: MaybeParser
+parseProlog = orNothing . parseProlog'
 
-parseProlog :: EitherParser ParseError
-parseProlog = fmap compact . parse program ""
+parseProlog' = fmap compact . parse program ""
```
Comment (faloi) [inline src/Language/Mulang/Parsers/Prolog.hs]: Could be inlined.

Comment (flbulgarelli) [inline src/Language/Mulang/Parsers/Prolog.hs]: It is uses twice

**Pull request merged**